devtools/platform-descriptor-json/src/main/resources/bundled-codestarts/project/quarkus/codestart.yml

```diff
@@ -15,6 +15,9 @@ output-strategy:
 language:
   base:
     shared-data:
+      supports:
+        dev-mode: true
+        native: true
```

Comment:
I don't think you need to explicitly set those since they are default no?

Reply:
Its more for documentation purposes.  If somebody doesn't ready the docs, they will see those flags there and know they exist (instead of recreating them).

Reply:
ok :)